src/ResourceManager/Compute/Commands.Compute/Extension/AzureDiskEncryption/GetAzureDiskEncryptionStatus.cs

```diff
@@ -58,6 +58,22 @@ public class GetAzureDiskEncryptionStatusCommand : VirtualMachineExtensionBaseCm
         [ValidateNotNullOrEmpty]
         public string Name { get; set; }
         
+        [Parameter(
+            Mandatory = false,
+            Position = 3,
```

Comment:
@vermashi nit: you can decide whether or not the positions are needed for the new parameters ☺️